apps/desktop/src/mods/ModBox.tsx

```diff
@@ -138,6 +141,21 @@ export function ModBox(props: IModBoxProps) {
     });
   };
 
+  const updated = (updated: string) => {
+    var diff =
+      (new Date().valueOf() - new Date(updated).valueOf()) / (1000 * 3600 * 24);
+    return diff < 1 ? 'today' : Math.ceil(diff) + ' days ago';
+  };
+
+  const isUpdatedSinceGameUpdate = (updated: string) => {
+    if (
+      Math.ceil(new Date(updated).valueOf() / 1000) > Number(LatestGameUpdate)
+    ) {
+      return true;
+    }
+    return false;
+  };
```

Comment:
This could be memoized to make it simpler & avoid re-renders in react:

```ts
const updatedInfo = useMemo(() => {
  if (!mod.fetched?.mod_updated || !latestGameUpdated) return null;
  // Do the same calculations as before, return an object like:
  // { description: "3 days ago", isUpdatedSinceGameUpdated: true }
}, [mod.fetched?.mod_updated, latestGameUpdated])
```